server/handlers.go

```diff
@@ -178,22 +161,20 @@ func (s *Server) handleAuthorization(w http.ResponseWriter, r *http.Request) {
 
 	if len(connectors) == 1 && !s.alwaysShowLogin {
 		for _, c := range connectors {
```

Comment:
We already checked that connectors length equals 1. We can use `connectors[0]` instead of iterating.
Do you mind if I ask you to fix it? :)

Reply:
No problem.

Reply:
Fixed.